content/en/docs/contribute/localization.md

```diff
@@ -57,8 +66,9 @@ equivalent fix by updating the localization you're working on.
 Please limit pull requests to a single localization, since pull requests that change
 content in multiple localizations could be difficult to review.
 
-Follow [Suggesting Content Improvements](/docs/contribute/suggest-improvements/) to propose changes to
-that localization. The process is very similar to proposing changes to the upstream (English) content.
+Follow [Suggesting Content Improvements](/docs/contribute/suggesting-improvements/)
+to propose changes to that localization. The process is very similar to proposing changes to the
+upstream (English) content.
```

Comment:
It's tricky but the old one https://kubernetes.io/docs/contribute/suggest-improvements/ is correct.
The new link https://kubernetes.io/docs/contribute/suggesting-improvements/ cannot be found.

Reply:
Okay. I'm gonna remove the 'slug' then.